script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -299,7 +306,7 @@ I'm pretty sure I remember Arcueid mentioning she was royalty.
 [sha:af454b01e1169d0fd86246e2b8a40c19635ca2f9]{
 -- Page 596, Offset 18068.
 -- 「んー、中にはいるかもね、わたしに似た特性を持つように作られた死徒が」
-"Hmm, it's possible that somewhere out there is a Dead Apostle that was created in my image."
+"Hmm, maybe somewhere among them there is a Dead Apostle created with the same kind of attributes as me."
```

Comment:
Prefer original unless it is Wrong for Reasons

Reply:
She's kind of talking about Altrouge, or maybe implying that they'd do something like Altrouge again.

The original phrasing makes it sound way too explicit. I always joke that Nasu infodumps are omniscient, but that's why you cant imply something which isn't originally implied, because the reader will assume a lot of the things said in them. "Somewhere out there is a Dead Apostle that was created in my image" essentially makes Arcueid say that yes: there's a Successor out there who was created with me as a model. 

But she's just saying that there maybe is a 'fake True Ancestor-ish Dead Apostle' out there, hence the わたしに似た特性を持つように作られた. 

Sometimes you have to be a little careful.